pkg/tracker/disputeChecker.go

```diff
@@ -91,7 +90,7 @@ func CheckValueAtTime(reqID uint64, val *big.Int, at time.Time) *ValueCheckResul
 	}
 }
 
-func (c *disputeChecker) Exec(ctx context.Context) error {
+func (c *disputeChecker) Exec(ctx context.Context, logger log.Logger) error {
```

Comment:
Let's add a constructor here so that we can have the component name. the same in all other places for each component.

This
```
func (c *disputeChecker) Exec(ctx context.Context, logger log.Logger){
	logger.
}
```
should change into
```
func NewDisputeChecker(logger log.Logger) *DisputeChecker{
	return &disputeChecker {
		logger:  log.With(logger, "component", "dispute checker")
	}
}

func (c *DisputeChecker) Exec(ctx context.Context){
	c.logger
}
``